bussiness-logic-module/src/main/java/edu/hillel/App.java

```diff
@@ -38,5 +40,76 @@ public static void main( String[] args )
 //        // Виклик методу біна
 //        System.out.println(itemService.getAllItems());
 
+
+        CategoryRepository singletonInstance = CategoryRepositoryTxtFilesStorageImpl.getSingletonInstance();

```

Comment:
Please test your functionality in test folder App and main method should be clean

Reply:
done